### DISCUSSION TITLE
How do I run tests?

### DISCUSSION THREAD

**Discussion by sanxiyn** [carbon-language/carbon-lang#1771] [Explorer]
I built Explorer with `bazel build //explorer` and it worked first time! That's great.

I see some tests under `explorer/testdata`, which I think is meant to be run automatically. How do I run these tests? I made some changes to Explorer and I want to check I didn't break anything.

Maybe this is obvious if you have used Bazel but I haven't and I couldn't find how.

Thanks.

**Comment by geoffromer** [comment]
`bazel test` is the command for running tests. It takes the same kind of arguments as `bazel run`, so for example `bazel test explorer/...` will run all the tests under `explorer`.
